notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    client = EfdClient('summit_efd')
</pre><p>I suggest we use the <code>client = vandv.efd.create_efd_client()</code>. This function makes it transparent wether you are at the summit/tts/usdf.   </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Alternatively, we can ask Angelo to have that implemented in <code>lsst_efd_client</code></p><p> </p>

Reply:
I didn't do this.  For new people just learning the systems, this requires them to find and install two more packages.  It's simpler the way it is.